docs/fundamentals/aggregation-builder.txt

```diff
@@ -0,0 +1,515 @@
+.. _laravel-aggregation-builder:
+
+===================
+Aggregation Builder
+===================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: code example, pipeline, expression
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to perform aggregations and construct
+pipelines by using the {+odm-short+} aggregation builder. The aggregation
+builder lets you use a type-safe syntax to construct a MongoDB
+**aggregation pipeline**.
+
+An aggregation pipeline is a data processing pipeline that sequentially
+performs transformations and computations on input data, then outputs the
```

Comment:
"input data" sounds like data that you send with the command. It think this working could be better: "data from the MongoDB database"